cli.js

```diff
@@ -75,7 +79,7 @@ if (cli.flags.init) {
 	return;
 }
 
-var api = new Api(cli.input, {
+var api = new Api(cli.input.length ? cli.input : arrify(conf.files), {
```

Comment:
Maybe you meant that `ava just-testing.js` always should take precedence if you want to quick test something and not bother having to update `package.json` :P.